ui/lib/kmip/index.js

```diff
@@ -17,6 +17,10 @@ module.exports = EngineAddon.extend({
     enabled: true,
   },
 
+  babel: {
+    plugins: [require.resolve('ember-concurrency/async-arrow-task-transform')],
```

Comment:
What is this?!

Reply:
😂 required to use the [async-arrow notation](https://ember-concurrency.com/docs/installation/) -- the other engines already had it but when I refactored the form in KMIP I got an error that it needed this transform

Reply:
Aha! Thank you for explaining 😂